config/rbac/role.yaml

```diff
@@ -5,51 +5,20 @@ metadata:
   name: manager-role
 rules:
 - apiGroups:
```

Comment:
Something is definitely wrong here - role generation from code comments seems no to be working.